blocks/init/src/Blocks/components/layout/docs/readme.mdx

```diff
@@ -0,0 +1,11 @@
+# Layout Component
+
+Component used to provide layout grid columns functionality to your project.
```

Comment:
```suggestion
A component that is used to provide a layout (grid or columns) functionality to your project.
```

This seems a bit more descriptive as towhat the component actually provides (a container wrapper basically, no?).